src/main/resources/db/changelog/changelog-init-tables.yaml

```diff
@@ -158,7 +158,7 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: password
-                  type: VARCHAR(30)
+                  type: VARCHAR(255)
```

Comment:
Pls, lets do this change in separate file

Reply:
thanks, fixed